comp/netflow/portrollup/portrollup.go

```diff
@@ -83,6 +115,26 @@ func (prs *EndpointPairPortRollupStore) Add(sourceAddr []byte, destAddr []byte,
 	prs.AddToStore(prs.newStore, srcToDestKey, destToSrcKey, sourceAddr, destAddr, sourcePort, destPort, prs.IsEphemeralFromKeys(srcToDestKey, destToSrcKey))
 }
 
+// addWithFixedSizeKey implements the new fixed-size key approach for IPv4 only
+func (prs *EndpointPairPortRollupStore) addWithFixedSizeKey(sourceAddr []byte, destAddr []byte, sourcePort uint16, destPort uint16) {
```

Comment:
This is the change I mentioned to use a fixed size key to avoid the multiple extraneous memory allocations when generating a string key and to reduce portRollup.Add() function runtime by ~33%.